cert-manager/config/manager/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+resources:
+- manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: docker.io/annadreal/keptn-cert-operator
+  newTag: "202212151671103985"
```

Comment:
this should probably point to the real image on ghcr

Reply:
does not matter that much, this is rewritten during each build, in the operator/scheduler we also have Thomases account :)